### ISSUE TITLE
Chessboard unresponsive, most black pieces rendered incorrectly

### ISSUE THREAD

**Issue by faiz-akhtar** [enzomanuelmangano/react-native-chessboard#2]
![Screenshot_2023-02-12-04-02-59-794_host exp exponent](https://user-images.githubusercontent.com/22586065/218283931-e0f8966d-4e5f-4ae1-b11e-2a3266af5873.jpg)

App.js file below:

```
import React from 'react';
import { View } from 'react-native';
import Chessboard from 'react-native-chessboard';

export default function App() {
  return (
    <View
      style={{
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
      }}
    >
      <Chessboard />
    </View>
  )
};
```

**Comment by faiz-akhtar** [comment]
Wrapping <Chessboard /> in  <GestureHandlerRootView></GestureHandlerRootView> fixes the gestures.

**Comment by faiz-akhtar** [comment]
Oddly enough, adding some random colour to the black pieces png files fixes the rendering problem. But then we end up with ugly pieces.
![Screenshot_2023-02-12-06-03-34-011_host exp exponent](https://user-images.githubusercontent.com/22586065/218287015-5a372616-7b45-4778-8e92-0b6369e0c64d.jpg)

**Comment by faiz-akhtar** [comment]
The UI rendering issue occurs only when dark background (night mode) is enabled in android. Maybe this is an android issue, or a React Native issue, not an issue with the library. Still would welcome any workaround if there is any, for dark mode.

**Comment by scha-ch** [comment]
@faiz-akhtar it feels like 2 subjects:

1. unresponsive pieces?

You have to install this library (like i  the example in the README.md:

`import { GestureHandlerRootView } from "react-native-gesture-handler";`

and wrap your `<View>` w8th it.

2. coloring (e.g. darkmode)

it looks like you are coloring the pueces yourself or your darkmode function does so. If you apply the params like mentioned in README.md only the Square color changes. Without having looked into it, I assume pieces are SVG and therefore styled using `fill` property.

Still an issue?

**Comment by faiz-akhtar** [comment]
@scha-ch 
1. The unresponsive pieces issue was resolved by wrapping with GestureHandlerRootView .
2. The coloring issue was not resolved last time I checked. But that was in February. I am no longer working on the project which needed this repository. However, it is easy to check for anyone building an app with this chessboard. Just enable night mode in android and see if the pieces render correctly. 

I am going to close this issue as it is too old and I haven't verified if it still exists.
